webdriver/tests/bidi/browsing_context/print/margin.py

```diff
@@ -105,14 +109,55 @@ async def test_margin_same_as_page_dimension(
     await bidi_session.browsing_context.navigate(
         context=top_context["context"], url=page, wait="complete"
     )
-    print_value = await bidi_session.browsing_context.print(
+
+    # This yields and empty content area: https://github.com/w3c/webdriver-bidi/issues/473
+    with pytest.raises(UnsupportedOperationException):
+        await bidi_session.browsing_context.print(
+            context=top_context["context"],
+            shrink_to_fit=False,
+            margin=margin,
+        )
+
+
+@pytest.mark.parametrize(
+    "margin",
+    [
+        {"top": 27.94 - ceil(inch_in_cm / inch_in_point)},
+        {"left": 21.59 - ceil(inch_in_cm / inch_in_point)},
+        {"right": 21.59 - ceil(inch_in_cm / inch_in_point)},
+        {"bottom": 27.94 - ceil(inch_in_cm / inch_in_point)},
+    ],
+    ids=[
+        "top",
+        "left",
+        "right",
+        "bottom",
+    ],
+)
+async def test_minimum_page_size_with_individual_margins(
```

Comment:
I think that we can remove the `individual` from the test name given that it's just about the minimum page size.

```suggestion
async def test_margin_minimum_page_size(
```

Not sure if it would make sense to also have a parameterized value that sets all margins to the minimum page size. I'm fine without that one.